package.json

```diff
@@ -6,16 +6,17 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "lerna run build",
-    "start": "yarn build && node ./scripts/build.serve.js",
-    "lint": "lerna run lint"
+    "start": "yarn clean && node ./scripts/serve.js",
+    "lint": "lerna run lint",
+		"clean": "rimraf ./packages/*/dist"
```

Comment:
each package should have own clean
here we just need to execute `lerna run clean`